lib/utils/pages/feed_utils.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:campus_app/pages/rubnews/news_entity.dart';
+import 'package:campus_app/pages/rubnews/widgets/feed_item.dart';
+import 'package:campus_app/utils/pages/presentation_functions.dart';
+import 'package:flutter/widgets.dart';
```

Comment:
Again here: In order to keep the order _dart, flutter, external packages, internal imports_ this should be moved down.
```suggestion
import 'package:flutter/widgets.dart';
import 'package:cached_network_image/cached_network_image.dart';

import 'package:campus_app/pages/rubnews/news_entity.dart';
import 'package:campus_app/pages/rubnews/widgets/feed_item.dart';
import 'package:campus_app/utils/pages/presentation_functions.dart';
```